examples/bcs_mean_field.py

```diff
@@ -10,9 +10,9 @@
 
 The Bogoliubov transformation can be readily implemented by
 applying quantum gates on vertical pairs of qubits, which takes the form
-|BCS⟩ = \prod_k (u_k + v_k c^\dag_{k,↑} c^\dag_{−k,↓}|vac⟩ where |vac⟩ is
-the vacuum state and u_k^2 = (1+ ξ_k/(ξ_k^2+Δ_k^2)^{1/2})/2 and v_k^2
-= (1 - ξ_k/(ξ_k^2+Δ_k^2)^{1/2})/2.
+$|BCS⟩ = \prod_k (u_k + v_k c^\dagger_{k,↑} c^\dagger_{−k,↓}|vac⟩$ where $|vac⟩$
```

Comment:
Missing parenthesis: 

```suggestion
$|BCS⟩ = \prod_k (u_k + v_k c^\dagger_{k,↑} c^\dagger_{−k,↓}|vac⟩)$ where $|vac⟩$
```